inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -1146,4 +1146,21 @@ CREATE TABLE `sort_cluster_config`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8mb4 COMMENT ='Sort cluster config table';
 
+-- ----------------------------
+-- Table structure for id_params
+-- ----------------------------
+DROP TABLE IF EXISTS `sort_id_params`;
+CREATE TABLE `sort_id_params`
```

Comment:
The table name is not correct.
The table is the relation between sort task and inlong datastream, not about ID parameters of sort task.

Reply:
it's more like a common field of all id params, some extension params will be defined in tables like sort_id_params_kafka_ext,  sort_id_params_pulsar_ext